R/typecheck.R

```diff
@@ -240,3 +240,14 @@ get_autoconverter = function(target) {
 
   NULL
 }
+
+# Check that a vector of tags are valid tags for `PipeOp's`.
+# See `mlr_reflections$pipeops$valid_tags` for a list of valid tags.
+# @param tags [`character`]: A list of tags.
+# @return a character vector of tasks
+assert_tag = function(tags) {
```

Comment:
I don't know if we really need this kind of function; it is called only once, and mlr3::Learner just uses `assert_subset(..., mlr_reflections$...)` in its place.